### DISCUSSION TITLE
Not show errors when selected value is empty on `onBlur` and instead show error on `onSubmit`

### DISCUSSION THREAD

**Discussion by arpitdalal** [edmundhung/conform#386] [Ideas]
Is there a way to do this now or would you like to add this feature?
Description:
> User tabs through the inputs and the errors are shown as the input was blurred without a valid value because of the `shouldValidate: "onBlur"` in `useForm` but I would like to skip validation on blur when the value is just null or empty string and instead validate on submit in this case. But if the user inputs an incorrect value like entering only 5 characters for a field that requires a minimum of 8 characters, in this case, the error should be shown on blur.

Please share your thoughts on this, thank you.

**Comment by Glazy** [comment]
I'd really like to see this added to Conform. I actually wanted to implement the same thing but couldn't find a way to hack around it even.

If @edmundhung is keen on the idea, and we can come up with a suitable API, I'd be happy to take a look at implementing this.
